access-common/src/main/java/edu/unc/lib/dl/ui/controller/LorisContentController.java

```diff
@@ -39,10 +46,9 @@
 
 //    @Autowired
```

Comment:
is there a reason this field is commented out?

Reply:
It will probably end up being used once we have loris functioning again, so I left it behind as a placeholder